projects/plugins/boost/app/assets/src/js/pages/index/index.tsx

```diff
@@ -79,17 +42,14 @@ const Index = ( { criticalCss }: IndexProps ) => {
 				'jetpack-boost'
 		  );
 
-	const [ { data: suggestRegenerate } ] = useSuggestRegenerate();
 	const premiumFeatures = usePremiumFeatures();
 
 	return (
 		<div className="jb-container--narrow">
 			<Module
 				slug="critical_css"
 				title={ __( 'Optimize Critical CSS Loading (manual)', 'jetpack-boost' ) }
-				onDisable={ () => setAlreadyResumed( false ) }
-				onEnable={ resume }
-				onMountEnable={ resume }
+				onEnable={ () => requestRegenerateCriticalCss() }
```

Comment:
Can it be just this?

```suggestion
				onEnable={ requestRegenerateCriticalCss }
```

Reply:
Thanks, done in d326e2b8d7